src/i18n/en.go

```diff
@@ -221,7 +221,7 @@ const (
 	VpnStatusDnsStatusUnset          = "dns isn't set, try `/path/to/zcli vpn start` command"
 	VpnStatusAdditionalInfo          = "additional info:"
 	VpnStatusDnsCheckError           = "we failed to check that dns is working correctly"
-	VpnStatusDnsNoCheckFunction      = "there is no function for dns check"
+	VpnStatusCheckInvalidAddress     = "invalid address"
```

Comment:
as a user I'd like to know what address you talk about, e.g. "invalid IPv6 / IPv4 address"